service/controller/resource/masters/create_check_flatcar_migration_needed.go

```diff
@@ -17,7 +18,7 @@ func (r *Resource) checkFlatcarMigrationNeededTransition(ctx context.Context, ob
 		return Empty, microerror.Mask(err)
 	}
 
-	legacyExists, err := r.vmssExistsAndHasActiveInstance(ctx, key.ResourceGroupName(cr), key.LegacyMasterVMSSName(cr))
+	legacyExists, err := r.vmssExistsAndHasActiveInstance(ctx, cr, key.ResourceGroupName(cr), key.LegacyMasterVMSSName(cr))
```

Comment:
Here and in some other similar cases, the last two parameters can now be removed (and calculated inside of the function), just IMO it's not worth it, as these helper functions are mostly added due to migration to Flatcar, so I would not touch those, as that will be cleaned up once all clusters have migrated.

When we clean up Flatcar migration code, this will get cleaned up as well.